plaid/api/payment_initiation.py

```diff
@@ -41,20 +41,24 @@ def list_recipients(self):
 
         return self.client.post('/payment_initiation/recipient/list', {})
 
-    def create_payment(self, recipient_id, reference, amount):
+    def create_payment(self, recipient_id, reference, amount, options=None):
```

Comment:
Is there a reason not to use options={} here instead of doing a check below?

Reply:
I did try this in an earlier commit but linter fails with `mutable default arg of type Dict`. https://github.com/plaid/plaid-python/pull/322/commits/ddd5dd92c1c9f57826fb3ce1f25f34bc128eb2ce#diff-c6b6813713e91eebd0245fdd44beff804f81cc475c8dacaf34590a3e98967ec5R44

If there's an alternative to the additional check I'm happy to try it out.